bin/reth/src/node/mod.rs

```diff
@@ -376,6 +377,7 @@ impl Command {
 async fn run_network_until_shutdown<C>(
     shutdown: reth_tasks::shutdown::Shutdown,
     network: NetworkManager<C>,
+    secret_key: &SecretKey,
```

Comment:
the network already has this so we don't actually need it if we let the network do it.